lib/rucio/web/rest/flaskapi/v1/accounts.py

```diff
@@ -917,66 +917,74 @@ def get(self, account: str, rse: str) -> Response:
-                      bytes:
-                        description: "The number of bytes used."
-                        type: integer
-                      bytes_limit:
-                        description: "The maximum number of bytes."
-                        type: integer
-                      bytes_remaining:
-                        description: "The remaining number of bytes."
-                        type: integer
-          401:
-            description: "Invalid Auth Token"
-          404:
-            description: "Account or rse not found"
-          406:
-            description: "Not acceptable"
-        """
-        try:
-            def generate(issuer: str, vo: str) -> "Iterator[str]":
-                for usage in get_local_account_usage(account=account, rse=rse, issuer=issuer, vo=vo):
-                    yield dumps(usage, cls=APIEncoder) + '\n'
-
-            return try_stream(generate(issuer=request.environ['issuer'], vo=request.environ['vo']))
-        except (AccountNotFound, RSENotFound) as error:
-            return generate_http_error_flask(404, error)
-        except AccessDenied as error:
-            return generate_http_error_flask(401, error)
+  @check_accept_header_wrapper_flask(['application/x-json-stream'])
+  def get(self, account: str, rse: Optional[str] = None) -> Response:
+    """
+    ---
+    summary: Get local account usage
+    description: "Returns the local account usage."
+    tags:
+    - Account
+    parameters:
+    - name: account
+      in: path
+      description: "The account identifier."
+      schema:
+        type: string
+      style: simple
+    - name: rse
+      in: path
+      description: "The rse identifier."
+      schema:
+        type: string
+      style: simple
+    - name: unique
+      in: query
+      description: "If true, count unique replicas to avoid double-counting when multiple locks exist."
+      schema:
+        type: boolean
+        default: false
+      style: simple
+    responses:
+      200:
+        description: "OK"
+        content:
+          application/x-json-stream:
+            schema:
+              type: array
+              items:
+                type: object
+                properties:
+                  rse_id:
+                    description: "The rse id."
+                    type: string
+                  bytes:
+                    description: "The number of bytes used."
+                    type: integer
+                  bytes_limit:
+                    description: "The maximum number of bytes."
+                    type: integer
+                  bytes_remaining:
+                    description: "The remaining number of bytes."
+                    type: integer
+      401:
+        description: "Invalid Auth Token"
+      404:
+        description: "Account or rse not found"
+      406:
+        description: "Not acceptable"
+    """
+    unique = request.args.get('unique', 'false').lower() == 'true'
```

Comment:
Other boolean args accept 0 and 1 as false and true
```suggestion
    unique = request.args.get('unique', 'false').lower() in ['true', "1"]
```

Reply:
Addressed in subsequent commits

---

However, according to the [OpenAPI specification](https://swagger.io/docs/specification/v3_0/data-models/data-types/#boolean) for boolean types:

> type: boolean represents two values: true and false. Note that truthy and 
> falsy values such as "true", "", 0 or null are not considered boolean values.

While accepting '1' and '0' maintains consistency with some existing endpoints, 
the OpenAPI spec is clear that booleans should be 'true' or 'false'. Since this 
is a new parameter, we could argue for strict spec compliance here.

Reply:
Thanks for doing the research, if that's the case we should make this consistent across the REST API.